lib/rucio/common/utils.py

```diff
@@ -723,3 +724,18 @@ def add_url_query(url, query):
     mod_query.update(query)
     url_parts[4] = urllib.urlencode(mod_query)
     return urlparse.urlunparse(url_parts)
+
+
+def extract_storage_unit_and_value(input_string):
```

Comment:
What I had in mind as moving all of the conversion part to the method, including the kb, mb, gb, etc. part. as it is really something we might want to use in different places as well.

Reply:
Okay, makes sense. Then I would add a function to get the bytes value from a string, e.g. get_bytes_from_string()